mars/services/scheduling/worker/execution.py

```diff
@@ -32,6 +32,12 @@
 from ...meta import MetaAPI
 from ...storage import StorageAPI
 from ...subtask import Subtask, SubtaskAPI, SubtaskResult, SubtaskStatus
+from ...tests.fault_injection_manager import (
```

Comment:
Test modules shall never be used in other part of the code.

Reply:
Is there better ways to inject fault to test the code?

Reply:
> Is there better ways to inject fault to test the code?

I think it is better to use `patch` module. Some work is needed to pass method patches to different processes.

Reply:
> > Is there better ways to inject fault to test the code?
> 
> I think it is better to use `patch` module. Some work is needed to pass method patches to different processes.

Can you give more information about the `patch` module?